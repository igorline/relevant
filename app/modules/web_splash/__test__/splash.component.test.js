import React from 'react';
import renderer from 'react-test-renderer';
import SplashComponent from 'modules/web_splash/splash.component';
import 'jest-styled-components';
import { MemoryRouter } from 'react-router-dom';

const props = {
  cta: 'SIGN_UP'
};

jest.mock('react-redux', () => {
  return {
    useDispatch: () => {},
    connect: el => el
  };
});

test('Snapshot SplashComponent', () => {
  const tree = renderer
    .create(
      <MemoryRouter>
        <SplashComponent {...props} overRideDismiss />
      </MemoryRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
