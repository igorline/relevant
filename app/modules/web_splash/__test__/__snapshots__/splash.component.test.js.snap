// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot SplashComponent 1`] = `
.c1 {
  color: #0000ff;
}

.c0 {
  color: #0000ff;
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

<div
  className="splashContent"
  style={
    Object {
      "WebkitAlignItems": "center",
      "WebkitBoxAlign": "center",
      "WebkitBoxDirection": "normal",
      "WebkitBoxOrient": "horizontal",
      "WebkitBoxPack": "start",
      "WebkitFlexBasis": "auto",
      "WebkitFlexDirection": "row",
      "WebkitFlexShrink": 0,
      "WebkitJustifyContent": "flex-start",
      "alignItems": "center",
      "borderBottomColor": "rgba(221,221,221,1.00)",
      "borderBottomStyle": "solid",
      "borderBottomWidth": "1px",
      "borderLeftStyle": "solid",
      "borderLeftWidth": "0px",
      "borderRightStyle": "solid",
      "borderRightWidth": "0px",
      "borderTopStyle": "solid",
      "borderTopWidth": "0px",
      "boxSizing": "border-box",
      "display": "flex",
      "flexBasis": "auto",
      "flexDirection": "row",
      "flexShrink": 0,
      "justifyContent": "flex-start",
      "marginBottom": "0px",
      "marginLeft": "0px",
      "marginRight": "0px",
      "marginTop": "0px",
      "maxHeight": "550px",
      "minHeight": "0px",
      "minWidth": "0px",
      "msFlexAlign": "center",
      "msFlexDirection": "row",
      "msFlexNegative": 0,
      "msFlexPack": "start",
      "msFlexPreferredSize": "auto",
      "overflowX": "hidden",
      "overflowY": "hidden",
      "paddingBottom": "0px",
      "paddingLeft": "0px",
      "paddingRight": "0px",
      "paddingTop": "0px",
      "position": "relative",
      "zIndex": 0,
    }
  }
>
  <div
    data-focusable={true}
    onKeyDown={[Function]}
    onKeyUp={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "WebkitAlignItems": "stretch",
        "WebkitBoxAlign": "stretch",
        "WebkitBoxDirection": "normal",
        "WebkitBoxOrient": "vertical",
        "WebkitFlexBasis": "auto",
        "WebkitFlexDirection": "column",
        "WebkitFlexShrink": 0,
        "alignItems": "stretch",
        "borderBottomStyle": "solid",
        "borderBottomWidth": "0px",
        "borderLeftStyle": "solid",
        "borderLeftWidth": "0px",
        "borderRightStyle": "solid",
        "borderRightWidth": "0px",
        "borderTopStyle": "solid",
        "borderTopWidth": "0px",
        "boxSizing": "border-box",
        "cursor": "pointer",
        "display": "flex",
        "flexBasis": "auto",
        "flexDirection": "column",
        "flexShrink": 0,
        "height": "2.1rem",
        "marginBottom": "0px",
        "marginLeft": "0px",
        "marginRight": "0px",
        "marginTop": "0px",
        "minHeight": "0px",
        "minWidth": "0px",
        "msFlexAlign": "stretch",
        "msFlexDirection": "column",
        "msFlexNegative": 0,
        "msFlexPreferredSize": "auto",
        "msTouchAction": "manipulation",
        "overflowX": "hidden",
        "overflowY": "hidden",
        "paddingBottom": "0px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "paddingTop": "0px",
        "position": "absolute",
        "right": "4.2rem",
        "top": "4.2rem",
        "touchAction": "manipulation",
        "width": "2.1rem",
        "zIndex": 10,
      }
    }
    tabIndex="0"
  >
    <div
      style={
        Object {
          "WebkitAlignItems": "stretch",
          "WebkitBoxAlign": "stretch",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "vertical",
          "WebkitFlexBasis": "auto",
          "WebkitFlexDirection": "column",
          "WebkitFlexShrink": 0,
          "alignItems": "stretch",
          "backgroundColor": "rgba(0,0,0,0.00)",
          "backgroundImage": "url(\\"x.png\\")",
          "backgroundPosition": "center",
          "backgroundRepeat": "no-repeat",
          "backgroundSize": "contain",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0px",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0px",
          "borderRightStyle": "solid",
          "borderRightWidth": "0px",
          "borderTopStyle": "solid",
          "borderTopWidth": "0px",
          "bottom": "0px",
          "boxSizing": "border-box",
          "display": "flex",
          "flexBasis": "auto",
          "flexDirection": "column",
          "flexShrink": 0,
          "height": "100%",
          "left": "0px",
          "marginBottom": "0px",
          "marginLeft": "0px",
          "marginRight": "0px",
          "marginTop": "0px",
          "minHeight": "0px",
          "minWidth": "0px",
          "msFlexAlign": "stretch",
          "msFlexDirection": "column",
          "msFlexNegative": 0,
          "msFlexPreferredSize": "auto",
          "paddingBottom": "0px",
          "paddingLeft": "0px",
          "paddingRight": "0px",
          "paddingTop": "0px",
          "position": "absolute",
          "right": "0px",
          "top": "0px",
          "width": "100%",
          "zIndex": -1,
        }
      }
    />
    <img
      alt=""
      draggable={false}
      src="x.png"
      style={
        Object {
          "bottom": "0px",
          "height": "100%",
          "left": "0px",
          "opacity": 0,
          "position": "absolute",
          "right": "0px",
          "top": "0px",
          "width": "100%",
          "zIndex": -1,
        }
      }
    />
  </div>
  <div
    className="mainSection"
    style={
      Object {
        "WebkitAlignItems": "flex-start",
        "WebkitBoxAlign": "start",
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 1,
        "WebkitBoxOrient": "vertical",
        "WebkitBoxPack": "center",
        "WebkitFlexBasis": "0px",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 1,
        "WebkitFlexShrink": 1,
        "WebkitJustifyContent": "center",
        "alignItems": "flex-start",
        "borderBottomStyle": "solid",
        "borderBottomWidth": "0px",
        "borderLeftStyle": "solid",
        "borderLeftWidth": "0px",
        "borderRightStyle": "solid",
        "borderRightWidth": "0px",
        "borderTopStyle": "solid",
        "borderTopWidth": "0px",
        "boxSizing": "border-box",
        "display": "flex",
        "flexBasis": "0px",
        "flexDirection": "column",
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
        "marginBottom": "0rem",
        "marginLeft": "8.4rem",
        "marginRight": "8.4rem",
        "marginTop": "8.4rem",
        "minHeight": "0px",
        "minWidth": "0px",
        "msFlexAlign": "start",
        "msFlexDirection": "column",
        "msFlexNegative": 1,
        "msFlexPack": "center",
        "msFlexPositive": 1,
        "msFlexPreferredSize": "0px",
        "paddingBottom": "0px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "paddingTop": "0px",
        "position": "relative",
        "zIndex": 0,
      }
    }
  >
    <div
      style={
        Object {
          "WebkitAlignItems": "stretch",
          "WebkitBoxAlign": "stretch",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "vertical",
          "WebkitFlexBasis": "auto",
          "WebkitFlexDirection": "column",
          "WebkitFlexShrink": 0,
          "alignItems": "stretch",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0px",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0px",
          "borderRightStyle": "solid",
          "borderRightWidth": "0px",
          "borderTopStyle": "solid",
          "borderTopWidth": "0px",
          "boxSizing": "border-box",
          "display": "flex",
          "flexBasis": "auto",
          "flexDirection": "column",
          "flexShrink": 0,
          "marginBottom": "0px",
          "marginLeft": "0px",
          "marginRight": "0px",
          "marginTop": "0px",
          "minHeight": "0px",
          "minWidth": "0px",
          "msFlexAlign": "stretch",
          "msFlexDirection": "column",
          "msFlexNegative": 0,
          "msFlexPreferredSize": "auto",
          "paddingBottom": "0px",
          "paddingLeft": "0px",
          "paddingRight": "0px",
          "paddingTop": "0px",
          "position": "relative",
          "zIndex": 0,
        }
      }
    >
      <div
        dir="auto"
        fs={
          Array [
            6,
            3,
          ]
        }
        lh={
          Array [
            9,
            4.2,
          ]
        }
        style={
          Object {
            "borderBottomWidth": "0px",
            "borderLeftWidth": "0px",
            "borderRightWidth": "0px",
            "borderTopWidth": "0px",
            "boxSizing": "border-box",
            "color": "rgba(0,0,0,1.00)",
            "display": "inline",
            "fontFamily": "HelveticaNeue-Medium",
            "fontSize": "4.2rem",
            "fontStyle": "inherit",
            "fontVariant": "inherit",
            "fontWeight": "inherit",
            "lineHeight": "6.3rem",
            "marginBottom": "0px",
            "marginLeft": "0px",
            "marginRight": "0px",
            "marginTop": "0px",
            "paddingBottom": "0px",
            "paddingLeft": "0px",
            "paddingRight": "0px",
            "paddingTop": "0px",
            "textDecoration": "none",
            "whiteSpace": "pre-wrap",
            "wordWrap": "break-word",
          }
        }
      >
        <span
          dir="auto"
          style={
            Object {
              "WebkitFlexBasis": "auto",
              "WebkitFlexShrink": 0,
              "borderBottomWidth": "0px",
              "borderLeftWidth": "0px",
              "borderRightWidth": "0px",
              "borderTopWidth": "0px",
              "boxSizing": "border-box",
              "color": "inherit",
              "display": "flex",
              "flexBasis": "auto",
              "flexShrink": 0,
              "fontFamily": "inherit",
              "fontSize": "inherit",
              "fontStyle": "inherit",
              "fontVariant": "inherit",
              "fontWeight": "inherit",
              "lineHeight": "inherit",
              "marginBottom": "0px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "msFlexNegative": 0,
              "msFlexPreferredSize": "auto",
              "paddingBottom": "0px",
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "paddingTop": "0px",
              "position": "relative",
              "textDecoration": "none",
              "whiteSpace": "inherit",
              "wordWrap": "break-word",
            }
          }
        >
          A new kind of social network built on trust.
        </span>
      </div>
      <div
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "vertical",
            "WebkitFlexBasis": "auto",
            "WebkitFlexDirection": "column",
            "WebkitFlexShrink": 0,
            "alignItems": "stretch",
            "borderBottomStyle": "solid",
            "borderBottomWidth": "0px",
            "borderLeftStyle": "solid",
            "borderLeftWidth": "0px",
            "borderRightStyle": "solid",
            "borderRightWidth": "0px",
            "borderTopStyle": "solid",
            "borderTopWidth": "0px",
            "boxSizing": "border-box",
            "display": "flex",
            "flexBasis": "auto",
            "flexDirection": "column",
            "flexShrink": 0,
            "marginBottom": "5.6rem",
            "marginLeft": "0px",
            "marginRight": "0px",
            "marginTop": "3.5rem",
            "minHeight": "0px",
            "minWidth": "0px",
            "msFlexAlign": "stretch",
            "msFlexDirection": "column",
            "msFlexNegative": 0,
            "msFlexPreferredSize": "auto",
            "paddingBottom": "0px",
            "paddingLeft": "0px",
            "paddingRight": "0px",
            "paddingTop": "0px",
            "position": "relative",
            "zIndex": 0,
          }
        }
      >
        <div
          dir="auto"
          fs={
            Array [
              2.5,
              1.5,
            ]
          }
          lh={
            Array [
              4,
              3,
            ]
          }
          style={
            Object {
              "borderBottomWidth": "0px",
              "borderLeftWidth": "0px",
              "borderRightWidth": "0px",
              "borderTopWidth": "0px",
              "boxSizing": "border-box",
              "color": "inherit",
              "display": "inline",
              "fontFamily": "Georgia",
              "fontSize": "1.75rem",
              "fontStyle": "inherit",
              "fontVariant": "inherit",
              "fontWeight": "inherit",
              "lineHeight": "2.8rem",
              "marginBottom": "0px",
              "marginLeft": "0px",
              "marginRight": "0px",
              "marginTop": "0px",
              "paddingBottom": "0px",
              "paddingLeft": "0px",
              "paddingRight": "0px",
              "paddingTop": "0px",
              "position": "relative",
              "textDecoration": "none",
              "whiteSpace": "pre-wrap",
              "wordWrap": "break-word",
            }
          }
        >
          Join a community, curate content and earn rewards.
           
          <a
            className="ulink-mock c0"
            display="inline"
            href="https://blog.relevant.community/relevant-beta-is-live-c385d0e1286c"
            onClick={[Function]}
            target="_blank"
          >
            Learn More
          </a>
        </div>
      </div>
    </div>
    <div
      style={
        Object {
          "WebkitAlignItems": "stretch",
          "WebkitBoxAlign": "stretch",
          "WebkitBoxDirection": "normal",
          "WebkitBoxOrient": "vertical",
          "WebkitFlexBasis": "auto",
          "WebkitFlexDirection": "column",
          "WebkitFlexShrink": 0,
          "alignItems": "stretch",
          "borderBottomStyle": "solid",
          "borderBottomWidth": "0px",
          "borderLeftStyle": "solid",
          "borderLeftWidth": "0px",
          "borderRightStyle": "solid",
          "borderRightWidth": "0px",
          "borderTopStyle": "solid",
          "borderTopWidth": "0px",
          "boxSizing": "border-box",
          "display": "flex",
          "flexBasis": "auto",
          "flexDirection": "column",
          "flexShrink": 0,
          "marginBottom": "0px",
          "marginLeft": "0px",
          "marginRight": "0px",
          "marginTop": "0px",
          "minHeight": "0px",
          "minWidth": "0px",
          "msFlexAlign": "stretch",
          "msFlexDirection": "column",
          "msFlexNegative": 0,
          "msFlexPreferredSize": "auto",
          "paddingBottom": "5.6rem",
          "paddingLeft": "0px",
          "paddingRight": "0px",
          "paddingTop": "0px",
          "position": "relative",
          "zIndex": 0,
        }
      }
    >
      <div
        style={
          Object {
            "WebkitAlignItems": "stretch",
            "WebkitBoxAlign": "stretch",
            "WebkitBoxDirection": "normal",
            "WebkitBoxOrient": "horizontal",
            "WebkitBoxPack": "start",
            "WebkitFlexBasis": "auto",
            "WebkitFlexDirection": "row",
            "WebkitFlexShrink": 0,
            "WebkitJustifyContent": "flex-start",
            "alignItems": "stretch",
            "borderBottomStyle": "solid",
            "borderBottomWidth": "0px",
            "borderLeftStyle": "solid",
            "borderLeftWidth": "0px",
            "borderRightStyle": "solid",
            "borderRightWidth": "0px",
            "borderTopStyle": "solid",
            "borderTopWidth": "0px",
            "boxSizing": "border-box",
            "display": "flex",
            "flexBasis": "auto",
            "flexDirection": "row",
            "flexShrink": 0,
            "justifyContent": "flex-start",
            "marginBottom": "0px",
            "marginLeft": "0px",
            "marginRight": "0px",
            "marginTop": "0px",
            "minHeight": "0px",
            "minWidth": "0px",
            "msFlexAlign": "stretch",
            "msFlexDirection": "row",
            "msFlexNegative": 0,
            "msFlexPack": "start",
            "msFlexPreferredSize": "auto",
            "paddingBottom": "0px",
            "paddingLeft": "0px",
            "paddingRight": "0px",
            "paddingTop": "0px",
            "position": "relative",
            "zIndex": 0,
          }
        }
      >
        <a
          className="ulink-mock c1"
          href="/user/login?redirect=/"
          hu={0}
          inline={0}
          onClick={[Function]}
          styles=""
        >
          <div
            dir="auto"
            mr={4}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxPack": "center",
                "WebkitFlexBasis": "auto",
                "WebkitFlexShrink": 0,
                "WebkitJustifyContent": "center",
                "alignItems": "center",
                "backgroundColor": "rgba(0,0,255,1.00)",
                "borderBottomWidth": "0px",
                "borderLeftWidth": "0px",
                "borderRightWidth": "0px",
                "borderTopWidth": "0px",
                "boxSizing": "border-box",
                "color": "rgba(255,255,255,1.00)",
                "cursor": "pointer",
                "display": "flex",
                "flexBasis": "auto",
                "flexShrink": 0,
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": "1.05rem",
                "fontStyle": "inherit",
                "fontVariant": "inherit",
                "fontWeight": "inherit",
                "height": "4.2rem",
                "justifyContent": "center",
                "lineHeight": "inherit",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "2.8rem",
                "marginTop": "0px",
                "minWidth": "12.6rem",
                "msFlexAlign": "center",
                "msFlexNegative": 0,
                "msFlexPack": "center",
                "msFlexPreferredSize": "auto",
                "paddingBottom": "0px",
                "paddingLeft": "1.4rem",
                "paddingRight": "1.4rem",
                "paddingTop": "0px",
                "position": "relative",
                "textDecoration": "none",
                "whiteSpace": "pre-wrap",
                "wordWrap": "break-word",
              }
            }
          >
            Login
          </div>
        </a>
        <a
          className="ulink-mock c1"
          href="/user/signup?redirect=/"
          hu={0}
          inline={0}
          onClick={[Function]}
          styles=""
        >
          <div
            dir="auto"
            mr={0}
            style={
              Object {
                "WebkitAlignItems": "center",
                "WebkitBoxAlign": "center",
                "WebkitBoxPack": "center",
                "WebkitFlexBasis": "auto",
                "WebkitFlexShrink": 0,
                "WebkitJustifyContent": "center",
                "alignItems": "center",
                "backgroundColor": "rgba(0,0,255,1.00)",
                "borderBottomWidth": "0px",
                "borderLeftWidth": "0px",
                "borderRightWidth": "0px",
                "borderTopWidth": "0px",
                "boxSizing": "border-box",
                "color": "rgba(255,255,255,1.00)",
                "cursor": "pointer",
                "display": "flex",
                "flexBasis": "auto",
                "flexShrink": 0,
                "fontFamily": "HelveticaNeue-Medium",
                "fontSize": "1.05rem",
                "fontStyle": "inherit",
                "fontVariant": "inherit",
                "fontWeight": "inherit",
                "height": "4.2rem",
                "justifyContent": "center",
                "lineHeight": "inherit",
                "marginBottom": "0px",
                "marginLeft": "0px",
                "marginRight": "0px",
                "marginTop": "0px",
                "minWidth": "12.6rem",
                "msFlexAlign": "center",
                "msFlexNegative": 0,
                "msFlexPack": "center",
                "msFlexPreferredSize": "auto",
                "paddingBottom": "0px",
                "paddingLeft": "1.4rem",
                "paddingRight": "1.4rem",
                "paddingTop": "0px",
                "position": "relative",
                "textDecoration": "none",
                "whiteSpace": "pre-wrap",
                "wordWrap": "break-word",
              }
            }
          >
            Sign Up
          </div>
        </a>
      </div>
    </div>
  </div>
  <div
    className="phone"
    style={
      Object {
        "WebkitAlignItems": "stretch",
        "WebkitAlignSelf": "flex-end",
        "WebkitBoxAlign": "stretch",
        "WebkitBoxDirection": "normal",
        "WebkitBoxFlex": 0.65,
        "WebkitBoxOrient": "vertical",
        "WebkitFlexBasis": "0px",
        "WebkitFlexDirection": "column",
        "WebkitFlexGrow": 0.65,
        "WebkitFlexShrink": 1,
        "WebkitTransformOrigin": "bottom",
        "alignItems": "stretch",
        "alignSelf": "flex-end",
        "borderBottomStyle": "solid",
        "borderBottomWidth": "0px",
        "borderLeftStyle": "solid",
        "borderLeftWidth": "0px",
        "borderRightStyle": "solid",
        "borderRightWidth": "0px",
        "borderTopStyle": "solid",
        "borderTopWidth": "0px",
        "boxSizing": "border-box",
        "display": "flex",
        "flexBasis": "0px",
        "flexDirection": "column",
        "flexGrow": 0.65,
        "flexShrink": 1,
        "marginBottom": "0px",
        "marginLeft": "0px",
        "marginRight": "0px",
        "marginTop": "0px",
        "minHeight": "0px",
        "minWidth": "0px",
        "msFlexAlign": "stretch",
        "msFlexDirection": "column",
        "msFlexItemAlign": "end",
        "msFlexNegative": 1,
        "msFlexPositive": 0.65,
        "msFlexPreferredSize": "0px",
        "paddingBottom": "0px",
        "paddingLeft": "0px",
        "paddingRight": "0px",
        "paddingTop": "0px",
        "position": "relative",
        "transformOrigin": "bottom",
        "zIndex": -1,
      }
    }
  >
    <img
      alt="phone"
      onLoad={[Function]}
      src="/img/hand-transparent.png"
      style={
        Object {
          "width": "100%",
        }
      }
    />
  </div>
</div>
`;
